pkg/minikube/bootstrapper/certs.go

```diff
@@ -197,3 +218,117 @@ func generateCerts(k8s config.KubernetesConfig) error {
 
 	return nil
 }
+
+func collectCACerts() (map[string]string, error) {
+	localPath := constants.GetMinipath()
+
+	isValidPem := func(hostpath string) (bool, error) {
```

Comment:
Do you mind moving this function out from collectCACerts? With nested functions it is more difficult to understand scope.

Reply:
No problem. I'll move it out.